AppController/djinn.rb

```diff
@@ -4494,18 +4519,24 @@ def start_app_dashboard()
     my_public = my_node.public_ip
     my_private = my_node.private_ip
 
+    # First commit the dashboard to the UserAppServer.
```

Comment:
Please change that to something along the lines of:

`"First reserve dashboard app ID."`
or
`"First store dashboard app ID in the database."`

The UserAppServer is just a level of indirection for that, if I recall correctly.